### PR TITLE
Test pull request

### PR DESCRIPTION
**Pull request by ElijahReva**
Fast inverse square root

### CODE REVIEW

File: MagicMath.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+public class MagicMath
+{	
+	public static void main(String args []){
+		float input = 1;
+		System.out.println(invSqrt(input));
+	}
+	
+	public static float invSqrt(float x) {
+		float xhalf = 0.5f*x;
```
Comment (Rolandw0w) [inline MagicMath.java]: xHalf, due to Java Code Conventions

File: MagicMath.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		i = 0x5f3759df - (i>>1);
+		x = Float.intBitsToFloat(i);
+		x = x*(1.5f - xhalf*x*x);
+		return x;
+	}
+	
+	public static double invSqrt(double x) {
+		double xhalf = 0.5d*x;
+		long i = Double.doubleToLongBits(x);
+		i = 0x5fe6ec85e7de30daL - (i>>1);
```
Comment (Rolandw0w) [inline MagicMath.java]: Unclear code. Ніхуя не поняв

File: MagicMath.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import java.lang.System;
```
Comment (zinenkovlad) [inline MagicMath.java]: What the fuck?

File: MagicMath.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+public class MagicMath
```
Comment (Rolandw0w) [inline MagicMath.java]: public class MagicMath {
...
}

**Comment by ElijahReva** [comment]
Hey mate, why so offensive?

**Pull request closed without merging**